src/runtime/methodbinder.cs

```diff
@@ -92,51 +95,54 @@ internal class MethodBinder {
          }
 
 
-		 //====================================================================
-		 // Given a sequence of MethodInfo and two sequences of type parameters, 
-		 // return the MethodInfo that matches the signature and the closed generic.
-		 //====================================================================
-
-		 internal static MethodInfo MatchSignatureAndParameters(MethodInfo[] mi, Type[] genericTp, Type[] sigTp)
-		 {
-			 int genericCount = genericTp.Length;
-			 int signatureCount = sigTp.Length;
-			 for (int i = 0; i < mi.Length; i++)
-			 {
-				 if (!mi[i].IsGenericMethodDefinition)
-				 {
-					 continue;
-				 }
-				 Type[] genericArgs = mi[i].GetGenericArguments();
-				 if (genericArgs.Length != genericCount)
-				 {
-					 continue;
-				 }
-				 ParameterInfo[] pi = mi[i].GetParameters();
-				 if (pi.Length != signatureCount)
-				 {
-					 continue;
-				 }
-				 for (int n = 0; n < pi.Length; n++)
-				 {
-					 if (sigTp[n] != pi[n].ParameterType)
-					 {
-						 break;
-					 }
-					 if (n == (pi.Length - 1))
-					 {
-						 MethodInfo match = mi[i];
-						 if (match.IsGenericMethodDefinition)
-						 {
-							 Type[] typeArgs = match.GetGenericArguments();
-							 return match.MakeGenericMethod(genericTp);
-						 }
-						 return match;
-					 }
-				 }
-			 }
-			 return null;
-		 }
+         //====================================================================
```

Comment:
I assume nothing's actually changed here? Just whitespace? It's easier to manage white space changes in their own PR (just for future reference, don't worry about changing it back)